src/Xamarin.Android.Tools.AndroidSdk/AndroidVersions.cs

```diff
@@ -173,6 +173,10 @@ static bool MatchesId (AndroidVersion version, string id)
 			new AndroidVersion (28, "9.0",   "Pie") {
 				AlternateIds = new[]{ "P" },
 			},
+			new AndroidVersion (29, "10.0",  "Android X") {
```

Comment:
"Android X" was the codeName for API-29?!

I don't remember that codename…

Reply:
I guess it is `Android10`?

https://source.android.com/setup/start/build-numbers